src/map/npc.cpp

```diff
@@ -6185,7 +6187,9 @@ void do_init_npc(void){
 	// process all npc files
 	ShowStatus("Loading NPCs...\r");
 	for( file = npc_src_files; file != NULL; file = file->next ) {
+#ifdef DEBUG
```

Comment:
Above the loop and after the loop the same messages as everywhere should be added.
Maybe harmonize the wording as well!
Cant suggest that on Github Web, sorry!